stream-chat-android-client/src/main/java/io/getstream/chat/android/client/uploader/FileUploader.kt

```diff
@@ -5,14 +5,16 @@ import java.io.File
 
 public interface FileUploader {
 
+    public fun setConnection(userId: String, connectionId: String)
+
     public fun sendFile(
         channelType: String,
         channelId: String,
         file: File,
         callback: ProgressCallback
     )
 
-    public fun sendFile(channelType: String, channelId: String, file: File): String
+    public fun sendFile(channelType: String, channelId: String, file: File): String?
```

Comment:
This interface would be a bit hard to implement for a client just by looking at this file. For example, what are these `send...` methods supposed to return? Why are there two `send...` methods? Would be nice to add KDoc comments for these.